src/main/java/duke/TaskType.java

```diff
@@ -0,0 +1,10 @@
+package duke;
+
+/**
+ * Task representation upon parsing input.
+ */
+public enum TaskType {
```

Comment:
Just wondering, why would we need an enm for TaskType? Thank you!

Reply:
I foresaw that I might need to implement more OOP features into the parser function since I found implementing a separate group of classes for commands, as suggested on the CS2103 website, a bit superfluous. Hence, I decided to use an enum for TaskType to determine the command type for the parser. (: